### ISSUE TITLE
What exactly is this?

### ISSUE THREAD

**Issue by vonj** [svenschnelle/teensyscsi#1]
What does it do?  USB to SCSI interface?

And what license is it?

**Comment by epitaxial1** [comment]
Guess the author can't even be bothered to tell us.

**Comment by elosha** [comment]
Yes, it is a DIY SCSI controller for USB connection. Some would say a USB-to-SCSI adapter. As far as I know it supports UAS (USB attached SCSI) and USB MSC (Mass Storage Class) protocols.
